ote/src/cljs/ote/app/controller/route.cljs

```diff
@@ -166,13 +177,16 @@
 
   InitRouteTimes
   (process-event [_ app]
-    (assoc-in app [:route ::transit/trips]
-              [{::transit/stop-times (vec (map-indexed
-                                           (fn [stop-idx {::transit/keys [arrival-time departure-time]}]
-                                             {::transit/stop-idx stop-idx
-                                              ::transit/arrival-time arrival-time
-                                              ::transit/departure-time departure-time})
-                                           (get-in app [:route ::transit/stops])))}]))
+    (-> app
+        (assoc-in [:route ::transit/trips]
+                  [{::transit/stop-times (vec (map-indexed
```

Comment:
Do we need to add `:service-calendar-idx 0` to the trip?
A trip needs both stop times and a service calendar idx when saved in the database.